src/tools/pontoon/teams_projects.md

```diff
@@ -2,69 +2,68 @@
-
-There is also a special Localization page that allows to access all resources for all projects enabled for a specific locale: `/locale/all-projects/all-resources`.
+Pontoon organizes localizations by locale/project pairs. Pontoon's structure reflects this with separate pages for [Teams](#teams) (locales), [Projects](#projects) and [Localizations](#localization-page).
 
 The following diagram shows how these pages are organized and connected to each other:
 
 ![Team and project pages flow](../../assets/images/pontoon/teams_projects/page_flow.png)
 
-## Team page
+## Teams
+
+### Teams page
 
-You can access a team’s page by selecting it from the [Teams page](https://pontoon.mozilla.org/teams/) or by appending `/YOUR_LOCALE_CODE` to the URL, e.g. `https://pontoon.mozilla.org/it` for Italian.
+The Teams page lists all locales that are enabled in Pontoon. It can be reached by clicking `Teams` from the navbar or by using the `/teams` URL (e.g. [pontoon.mozilla.org/teams](https://pontoon.mozilla.org/teams/)). Selecting a specific [Team page](#team-page) shows a list of all projects enabled for the requested locale. Choosing one of these projects leads to the [Localization page](#localization-page).
 
-This is a screenshot of the [Italian team page](https://pontoon.mozilla.org/it):
+### Team page
 
-![Team page for Italian](../../assets/images/pontoon/teams_projects/team_page.png)
+You can access a specific team’s page by selecting it from the *Teams page* or by appending a locale code to the end of the Pontoon base URL with `/{LOCALE}` (e.g [pontoon.mozilla.org/it](https://pontoon.mozilla.org/it) for Italian).
 
-The page header contains linguistic information about the current locale (plural form, writing direction, etc.), plus an indication of the overall status of completion and statistics. The statistics displayed in the rightmost column are active links to access the Localization page for all projects, for example to reach all missing strings across projects available for Italian in Pontoon.
+The page header contains linguistic information about the current locale (plural form, writing direction, etc.), plus an indication of the overall status of completion and statistics. The statistics displayed in the rightmost column are active links to access the Localization page for all projects. For example, select `MISSING` to see all missing strings across projects enabled for that locale in Pontoon.
 
 ![Header of team page for Italian](../../assets/images/pontoon/teams_projects/team_page_header.png)
 
-There are 4 tabs available (highlighted in red), 5 if you’re an *admin* or *manager*:
+There are 5 tabs available to all roles, or 6 tabs (highlighted in red) for those with *admin* or *manager* roles:
```

Comment:
```suggestion
There are 5 tabs available to all roles, and 1 additional tab (highlighted in red) for those with *Administrator* or *Team manager* roles:
```

See http://localhost:3000/tools/pontoon/users.html#user-roles

Same note applies to the project page section.